packages/lab/src/pagination/Pagination.tsx

```diff
@@ -20,23 +20,23 @@ export interface PaginationProps extends HTMLAttributes<HTMLElement> {
   page?: number;
   initialPage?: number;
   onPageChange?: (page: number) => void;
-  compact?: boolean;
+  compact?: "default" | "goto";
```

Comment:
this prop name still sounds boolean, should it be renamed too? `compact=goto` is not very informative in what it's doing

Reply:
I agree. I'm still thinking of a better way to config this. Open to suggestions

Reply:
I've done some refactoring. Have another look when you get some time. Thanks ;)